app/views/settings/exports/show.html.haml

```diff
@@ -9,19 +9,19 @@
         %td= number_to_human_size @export.total_storage
         %td
       %tr
-        %th= t('accounts.posts', count: @export.total_statuses)
+        %th= t('accounts.posts.other')
         %td= number_with_delimiter @export.total_statuses
         %td
       %tr
-        %th= t('exports.follows')
```

Comment:
Why replace a string that seems to exist specifically for this page, though?

Reply:
It's kind of looks off with that “You” prefix being alongside with the ones above and below. And for prefix in general, I think it's redundant because the page itself is suggesting that it's your data to export, probably can remove it from the ones below too.